packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss

```diff
@@ -109,7 +111,6 @@
 }
 
 .form-container {
-  font-size: 40px !important;
```

Comment:
@denniskigen I was particularly interested in this line which leads to ginormous text

![Screen Shot 2022-08-16 at 6 39 13 AM](https://user-images.githubusercontent.com/5445264/184893765-1718de8c-7bce-47d9-b94f-edcb5861d67c.png)

Reply:
@donaldkibet , thanks for taking a look. Do you know anyone from your team or the community who would be appropriate to take this on?